### DISCUSSION TITLE
Thanks Dendraspis for the tip that black turning gray WAS caused by --colorrange full

### DISCUSSION THREAD

**Discussion by chowderhead** [staxrip/staxrip#1217] [General]
switching to colorrange limited made the blacks black again.
Thanks for the tip.  I had no idea.  Is there an easy way to tell if color is limited versus full??
Randy

**Comment by Dendraspis** [comment]
No problem.
I don't think you can *see* it, but you can use MediaInfo to read it out.

**Comment by chowderhead** [comment]
and there it is.  BOTTOM LEFT CORNER.  ok thats VERY useful thankyou.  Youve helped my color knowledge increase

.
![image](https://github.com/staxrip/staxrip/assets/22981743/b3723bab-3ecb-420d-9870-b04eac6f61af)

**Comment by chowderhead** [comment]
Im just curious dendraspis but are most modern movies made in FULL color does that mean 8 16 or 24 or 32 bit color?
Iworked on a ot of anime stuff restoring old anime videos to 1080p conversin but all are 8 bit color I think.  I NEVER REALLY CHECKED.  i mean have I ever actually seen a 16-24 or 32 bit movie.  or all of them just 10 bit color 1080-10p.  It feels a bit confusing.  Is high bit color mostly still images while video is at best 1080-10p these days.

**Comment by chowderhead** [comment]
Ive done a bit of reading.  So video is either 8 bits PER RGB CHANNEL or 10 which is really 24bit or 30bit data rendition. and then theres 12 bit hollywood video which is 36 bit video.  Tech cant even broadcast that much color info.  makes more sense now.  I didnt understand per channel is why 8biit is 16.7 million colors and not 256 and 10 bit is 1 billion colors.  ahh internet iis amazing.
